python/lsst/ts/ATDomeTrajectory/config_schema.py

```diff
@@ -1,4 +1,31 @@
-$schema: http://json-schema.org/draft-07/schema#
+# This file is part of ts_ATDomeTrajectory.
+#
+# Developed for the LSST Telescope and Site Systems.
```

Comment:
Please update this and all other files to Vera C. Rubin Observatory.

Reply:
Good catch. Done.